docs/pages/distribution/runtime-versions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Runtime Versions
+---
+
+> Custom runtime versions are not supported on the classic build system (`expo build`); these apps will always use the SDK version as the basis for determining runtime compatibility. 
+
+Over-the-air updates with `expo-updates` will only work in binaries with a [compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. By default, the runtime version is the Expo SDK version: the runtime is the set of libraries in the Expo SDK. Apps built with EAS Build will include only the dependencies that are present at the time that the app binary is built, and so the Expo SDK version will often not properly describe the runtime of the app. In these cases, you can start using `runtimeVersion` and it will take precedence over the Expo SDK version.
+
+A runtime version must conform to this [description](../versions/latest/config/app/#runtimeversion)
```

Comment:
Right now, this link does not display the regex we use to validate the runtime version.

I put up a PR to add a description field here: https://github.com/expo/universe/pull/7959

How do I update the local XDL with this update? Is there a script?

Reply:
Yes: `yarn run schema-sync <SDK version integer>`. Will run once https://github.com/expo/universe/pull/7959 lands.